src/Nova/Cart.php

```diff
@@ -30,6 +30,15 @@ class Cart extends BaseCheckoutResource
         \Tipoff\Locations\Nova\Filters\Location::class,
     ];
 
+    public static function indexQuery(NovaRequest $request, $query)
+    {
+        if ($request->user()->hasPermissionTo('all locations')) {
+            return $query;
+        }
```

Comment:
This is an example of using the new permission to determine whether or not the index results should be location filtered.  Ends up being much easier to manage than the list of explicit role names.

Reply:
Yes, this looks great. I'll make sure that gets added everywhere.

Reply:
With this change rolling out, this test template for index location filtering - https://github.com/tipoff/checkout/blob/b34d98f22cbd0ff435243ee0872eaf89f9b0187a/tests/Feature/Nova/CartItemResourceTest.php#L24 - should change as well.   Just need to test proper filtering for a user that does / doesn't have the permission now.